pxr/usd/usdUtils/sparseValueWriter.h

```diff
@@ -279,10 +279,24 @@ class UsdUtilsSparseValueWriter {
     }
 
     USDUTILS_API
-    const _AttrValueWriterMap& GetAttrValueWriterMap() const {
+    _AttrValueWriterMap GetAttrValueWriter() const {
```

Comment:
Thanks @HamedSabri-adsk  -- I think @spiffmon (and please correct me here, Spiff) meant that we'd like to not expose _AttrValueWriterMap (a private type) in the public API at all. Folks can now use GetSparseAttrValueWriters that you provided (that's nice, I think!), and the other suggestion was to maybe add an accessor that can provide a single UsdUtilsSparseAttrValueWriter given a name.